src/UI/Components/Sidebar.php

```diff
@@ -360,7 +360,7 @@ public function getOutput()
                                 echo "<tr id='messageWall".$pos."' style='z-index: 1;'>";
                                 echo "<td style='font-size: 95%; letter-spacing: 85%;'>";
                                 //Image
-                                $style = "style='width: 45px; height: 60px; float: right; margin-left: 6px; border: 1px solid black'";
+                                $style = "style='width: 45px; height: 50px; float: right; margin-left: 6px; border: 1px solid black'";
```

Comment:
Was the height change here intentional? Since we're working with a width-related bug, it doesn't look like the height should have changed.
```suggestion
                                $style = "style='width: 45px; height: 60px; float: right; margin-left: 6px; border: 1px solid black'";
```